docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -32,7 +32,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/romano/miniconda3/envs/default/lib/python3.5/site-packages/matplotlib/__init__.py:1350: UserWarning:  This call to matplotlib.use() has no effect\n",
+      "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/matplotlib/__init__.py:1357: UserWarning:  This call to matplotlib.use() has no effect\n",
```

Comment:
Do you want to add your new `MDGXS` types to this Notebook? Specifically, the text under Cell 18 has a list of all of the types supported by a `Library`, perhaps add your new ones there?